src/main/java/org/akaza/openclinica/controller/ScheduledExtractController.java

```diff
@@ -50,10 +50,10 @@
 
 
     @ApiOperation(value = "To get latest scheduled extract dataset ids and creation time for the job name at study level", notes = "only work for authorized users with the right access permission")
-    @RequestMapping(value = "/studies/{studyOID}/extractJobs/{jobName}/jobExecutions", method = RequestMethod.GET)
+    @RequestMapping(value = "/studies/{studyOID}/extractJobs/{jobUuid}/jobExecutions", method = RequestMethod.GET)
```

Comment:
Since the jobUuid is unique, I don't think we need to force the user to provide studyOID as well. I would update the url to "extractJobs/{jobUuid}/jobExecutions"